en/docs/learn/analytics/configuring-keystores-in-apim-analytics.md

```diff
@@ -1,175 +1,205 @@
+    | **keyStoreFile**       | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
     | **keyStorePassword**   | Keystore password                                                |
     | **certPass**           | Private key password                                             |
    
-3. **Securevault configuration**
-   
-    This is used to store encrypted passwords that are mapped to aliases.
-    ```yaml
-       wso2.securevault:
-         secretRepository:
-           ...
-           parameters:
-             privateKeyAlias: wso2carbon
-             keystoreLocation: ${sys:carbon.home}/resources/security/securevault.jks
-             ... 
-    ```
-    The elements in the above configuration are described below:
-      
-    | Element                | Description                                                      |
-    |------------------------|------------------------------------------------------------------|
-    | **privateKeyAlias**    | Private key password                                                |
-    | **keyStoreLocation**   | Name of the keystore file. **The file must be located in `<API-M_ANALYTICS_HOME>/resources/security`** |
+3. Update the Secure Vault configurations.
    
+     1. Update the configurations in the `deployment.yaml` file.
+    
+         The Secure Vault configurations are used to store encrypted passwords that are mapped to aliases.
 
-## Configuring Keystores in Worker Profile
+         ```yaml
+         wso2.securevault:
+           secretRepository:
+             ...
+             parameters:
+               privateKeyAlias: wso2carbon
+                keystoreLocation: ${sys:carbon.home}/resources/security/securevault.jks
+               ... 
+         ```
+        The elements in the above configuration are described below:
+          
+        | Element                | Description                                                      |
+        |------------------------|------------------------------------------------------------------|
+        | **privateKeyAlias**    | Private key alias                                                |
+        | **keyStoreLocation**   | Name of the keystore file. **The file must be located in the `<API-M_ANALYTICS_HOME>/resources/security` directory.** |
+
+    2. Provide the keystore and private key password in the Base64 encoded format in the `<ANALYTICS_HOME>/conf/dashboard/master-keys.yaml` file.
+
+        ??? info "Click here for more info on the `master-keys.yaml` file"
+            The base64 encoded plaintext keystore password and the base64 encoded plaintext private password of the keystore used for Secure Vault is defined in the master-keys.yaml file. WSO2 Carbon Server reads this file to obtain these two passwords, so that it can decrypt the passwords in the `secrets.properties` file in order to start the server.
+
+4. Set the correct keystore password.
```

Comment:
Let's add the configuration of the Keystore name in here as well.
```suggestion
4. Set the correct Keystore name.
    If you have created a new Keystore, then define the correct path for the new Keystore in the `-Djavax.net.ssl.keyStore` system property in the `<ANALYTICS_HOME>/wso2/dashboard/bin/carbon.sh` file when you change the Keystore.
```
After this, we can add the changing password step. Please do the same for the worker instructions(in this file) as well. Also please do the same change for the 3.1.0 version and 3.0.0 versions as well.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/1231/commits/f872500a49ef0c6223fab16659a66f2e133faa22